docs/en/observability/synthetics-projects-workflow.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[synthetics-project-monitors]]
+// lint ignore params
+= Working with project monitors
+
+++++
+<titleabbrev>Working with project monitors</titleabbrev>
+++++
+
+beta[]
+
+Projects are the most powerful and sophisticated way to configure synthetic monitors in the Elastic Stack. Projects let you define your infrastructure as code, more commonly known as IaaC or git-ops. With project monitors you organize your YAML configuration and Javascript/Typescript defined monitors on the filesystem, use Git for version control, and deploy via a CLI tool, usually executed on a CI/CD platform.
+
+
+== Getting started
+
+Working with projects requires working with the Elastic Synthetics CLI tool, which
+can be invoked via the `npx @elastic/synthetics` command. Before getting started
+you'll need to install Node.js; the `npx` command is provided by node, downloading
+the `@elastic/synthetics` NPM module if not yet available on your system.
+
+Run `npx @elastic/synthetics -h` to display the help menu and confirm your
+system is setup correctly
+
+It is highly recommended to use an https://cloud.elastic.co/[Elastic Cloud] deployment with this tutorial.
+If you wish to use a self-managed deployment, please read the <<synthetics-private-location>> docs first, and setup a private location with Fleet before proceeding.
+
+We'll start by creating our first project, run the command below to create a new 
+project named `projects-test` in the current directory.
+
+`npx @elastic/synthetics init projects-test`
+
+Then, follow the prompts on screen to setup the correct default variables for your project.
+When complete, set the `SYNTHETICS_API_KEY` environment variable in your terminal, which is used
+for authentication with your Elastic Stack. You will most likely want to set this permanently. 
+This is done differently for https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.2#saving-changes-to-environment-variables[Powershell] and https://unix.stackexchange.com/a/117470[Bash].
+
+Let's pause for a moment and examine the key files and directories inside your project. 
+
+* `journeys` contains the `.yaml`, `.ts` and `.js` files defining your monitors.
+* `synthetics.config.ts` contains settings for your project
+* `package.json` contains NPM settings for your project. Learn more in the https://docs.npmjs.com/about-packages-and-modules[NPM documentation].
+* `.github` contains sample Github actions workflow files for use with Github Actions.
```

Comment:
Use second person.

Also, what do you think about being explicit about the sample monitor code included when you create a new project?  We _could_ also add a link to [Synthetic tests configuration](https://www.elastic.co/guide/en/observability/current/synthetics-configuration.html) to "learn more" (like you did for "NPM documentation"), _but_ I'm not sure you want to encourage the reader to click over there just yet.

```suggestion
Then, take a look at key files and directories inside your project:

* `journeys` is where you'll add `.yaml`, `.ts`, and `.js` files defining your monitors. When you create a new project, this directory will contain files defining sample monitors.
* `synthetics.config.ts` contains settings for your project. When you create a new project, it will contain some basic configuration options that you can customize later.
* `package.json` contains NPM settings for your project. Learn more in the https://docs.npmjs.com/about-packages-and-modules[Synthetic tests configuration].
* `.github` contains sample Github actions workflow files for use with Github Actions.
```

Reply:
I agree, I'd rather not sidetrack the user here, and they can easily get there by clicking around.